pkg/groupsync/manytomanysyncer.go

```diff
@@ -83,6 +91,11 @@ func (f *ManyToManySyncer) Sync(ctx context.Context, sourceGroupID string) error
 		// get all source group IDs associated with the current target GroupID
 		sourceGroupIDs, err := f.targetGroupMapper.MappedGroupIDs(ctx, targetGroupID)
 		if err != nil {
+			logger.WarnContext(ctx, "error getting one ore more source group IDs for target group ID",
```

Comment:
Should these be warning or error?

Reply:
It could decide. these are errors where teamlink tries to 'keep going' with the sync process. So that made bias towards categorizing it as a Warning. What do you think?

Reply:
If the flow control continues, then warning probably makes sense. But if the flow continues, warning makes sense.

Reply:
Ok, then this is a warn